Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -296,8 +297,9 @@ class Detector : public o2::Base::Detector
     UInt_t *mChipTypeID;           //! Vector of detector type id
     Int_t *mBuildLevel;            //! Vector of Material Budget Studies
 
-    /// Container for data points
-    TClonesArray *mHitCollection;
+    /// Container for hit data
+    // TClonesArray *mHitCollection;
+    std::vector<o2::ITSMFT::Hit>* mHits;
```

Comment:
Should not this be std::unique_ptr<std::vector<o2::ITSMFT::Hit>> mHits; ?
Though we could avoid the pointer and simply have a vector data member

Reply:
I agree with either suggestion (which is the final goal) .. but for the moment neither works with FairRootManager::RegisterAny. So I am afraid we have to postpone this improvement.